doc/api/stream.md

```diff
@@ -1126,7 +1126,7 @@ buffer will be returned.
 If the `size` argument is not specified, all of the data contained in the
 internal buffer will be returned.
 
-The `size` argument must be less than or equal to 1 GB.
+The `size` argument must be less than or equal to 1 GiB.
```

Comment:
Not a blocker, as if anyone has any great ideas, they can always be implemented later, but I think the distinction is esoteric enough that we should spell it out on the first occurrence (or maybe even every occurrence). But I'm also having a hard time figuring out a way to do it in a straightforward way. `1024^3 bytes` is not exactly immediately clarifying. (My thought would be, "Isn't that the same as a GB?")